creativecloud/blocks/privacy-button/privacy-button.js

```diff
@@ -0,0 +1,48 @@
+import { getLibs, createTag } from '../../scripts/utils.js';
+
+function injectTrackingImage(url) {
+  const img = createTag('img', {
+    src: url,
+    class: 'privacy-tracking-image',
+    'aria-hidden': 'true',
+  });
+  document.body.appendChild(img);
+}
+
+function handleUrlVariant(button, urls, message) {
+  button.addEventListener('click', () => {
+    urls.split(',').forEach((url) => {
+      const trimmedUrl = url.trim();
+      if (trimmedUrl.match('^http') !== null) {
+        injectTrackingImage(trimmedUrl);
+      }
+    });
+    // Create a new element for the success message
+    button.parentNode.replaceChild(createTag('span', { class: 'privacy-button-message' }, message), button);
+  });
+}
+
+export default async function init(el) {
+  const miloLibs = getLibs('/libs');
+  const { decorateButtons } = await import(`${miloLibs}/utils/decorate.js`);
+  const config = {};
+  el.querySelectorAll(':scope > div').forEach((row) => {
+    const [key, value] = [...row.children];
+    if (key && value) {
+      config[key.textContent.trim().toLowerCase()] = value.textContent.trim();
+    }
+  });
+
+  const button = createTag('button', {
+    class: 'con-button blue',
+    type: 'button',
+  }, config['button-label']);
+
+  if (config['button-label'] && config['confirmation-message'] && config.url) {
```

Comment:
In case of having these keys in the authoring, would it be possible to have row 1 2 and 3 specifically tied to only these attributes and the author would not need to author a block with these keywords?

Reply:
Yes, it can be done. However, after discussing with the team, we finalised a default authoring structure where labels are placed in one column with corresponding values in the adjacent column. This approach makes it easier for authors to add values directly next to the corresponding labels.
@bandana147  @sonawanesnehal3 — let me know your thoughts on this.
<img width="641" alt="Screenshot 2024-11-26 at 6 52 55 PM" src="https://github.com/user-attachments/assets/a7f27a6d-d9f2-4002-a35c-0f8a62e53755">

Reply:
@aishwaryamathuria 
We proposed both authoring structures to Kyung Lee (author), and she prefers the current one. So, let's proceed with that.

Thanks!
cc: @bandana147

Reply:
@akanshaa-18 Since this has a key label format this would also need to be updated with the localization config to prevent the first column to be forwarded for localization int he HTML flow. Could you create a ticket for the same?

Reply:
Sure, will do that.
Thanks @aishwaryamathuria